build.zig

```diff
@@ -555,6 +560,13 @@ fn link_tracer_backend(
                 exe.linkSystemLibrary("dbghelp");
                 exe.linkSystemLibrary("ws2_32");
             }
+
+            // We might need to clone Tracy, if it doesn't exist
```

Comment:
Per TigerStyle, comments are sentences and end with a full stop or colon (same for the JUI comment below):

```suggestion
            // We might need to clone Tracy, if it doesn't exist.
```